README.md

```diff
@@ -12,11 +12,19 @@ https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder/
 
 ### Usage with a module bundler
 
-This module exports a single class called MapboxGeocoder as its default export,
-so in browserify or webpack, you can require it like:
+```bash
+npm install --save @mapbox/mapbox-gl-geocoder
+```
 
 ```js
-var MapboxGeocoder = require('@mapbox/mapbox-gl-geocoder');
```

Comment:
some people using broweserify would still need to use require, but I think for a modern usage example import is okay.